src/posts/getting-started-talkback.md

```diff
@@ -0,0 +1,74 @@
+- Accessibility Menu
+- Select to Speak
+- Switch Access
+
+The Android Accessibility Suite comes pre-installed on devices running Android 9 or greater.
+
+This article is an introduction to TalkBack version 13.1. We will walk you through the basics of enabling the software and navigating websites and applications with it.
+
+## Getting Started
+
+There are three ways to activate Talkback:
+1. From the settings menu: Accessibility > TalkBack > Toggle On
+2. On the physical device: Press and hold both volume keys for three seconds
+3. Via the Google Assistant: "Hey, Google. Turn on TalkBack"
+
+Once TalkBack is on, it will begin speaking instructions to you. While there are more robust features to explore, the basic commands you should know are:
+
+- Tapping the screen selects an item
+- Double tapping the screen activates your selection
+- When using a gesture, TalkBack requires you use 2 fingers instead of one
+
+## Navigation Modes
+
+There are at least three recognized ways to navigate with TalkBack: touch, swipe, and voice commands.
+
+### Navigating by touch
+
+To navigate by touch, touch anywhere on the screen, selecting any element. It will be highlighted by a green border box similar to what you would find when using a keyboard to navigate a web page. With pressure still applied, slowly drag your finger around the screen. As you move around the screen, TalkBack will jump to the next available item and announce its contents aloud. Once you've found the content you'd like to interact with, double-tap the screen.
+
+### Navigting by swiping
+
+Navigating by swiping can provide a more precise experience. To navigate in this way, select an initial item by tapping on the screen. Then, with one finger, move up and down the screen in a vertical motion. Items will be read aloud by TalkBack in the order they appear. Once you've found the content you'd like to interact with, double-tap the screen.
+
+### Navigating by voice command
+
+In addition to touch controls, TalkBack supports some voice commands. To activate voice commands, using one finger, swipe down and right. This will open the TalkBack Menu. Select voice commands from the list to open a prompt with more information about specific commands you can use to navigate the page with just your voice.
+
+## Basic TalkBack Commands
+
+Gesture | Description
+--------|------------
+Use one finger to tap your device's screen | Reads the selected item aloud
+Use two fingers to tap your device's screen | Pauses the current reading
+Double-tap your devices screen | Activates the selected item
+Move two fingers up or down the screen | Scrolls up or down the content
+Quickly flick up or down | Changes the read aloud speed
+Use one finger to swipe down and right | Opens the TalkBack Menu
+
+You can start using TalkBack and your new skills right now! Get a feel for how these commands work in real life by navigating [A11yProject.com](https://www.a11yproject.com/) with TalkBack turned on.
```

Comment:
```suggestion
## Wrapping up

You can start using TalkBack and your new skills right now! Get a feel for how these commands work in real life by navigating [A11yProject.com](https://www.a11yproject.com/) with TalkBack turned on.
```

Suggesting a final heading here, since the conclusion speaks to the entire piece, and not just the basic command table.